jaeger-thrift/build.gradle

```diff
@@ -6,6 +6,8 @@ description = 'Library to send data to Jaeger backend components via Thrift'
 dependencies {
     compile project(':jaeger-core')
 
+    compile group: 'org.slf4j', name: 'slf4j-api'
```

Comment:
Shouldn't it be `compileOnly`? Try to run a `mvn dependency:tree` (or `./gradlew dependencies`, if that's what your consumer application uses) and check if this version of `jaeger-thrift` (or `jaeger-client`) brings this dependency.